app/ai/voice/agents/automatic/services/mem0/memory.py

```
@@ -22,12 +22,11 @@
 
 try:
     from mem0 import Memory  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)